oshc/oshc/settings.py

```diff
@@ -24,6 +24,9 @@
 
 ALLOWED_HOSTS = ['*']
 
+# Tuple of people who get error notifications
+ADMINS = [('Tapasweni Pathak','tapaswenipathak@gmail.com'),('Nikhita Raghunath','nikitaraghunath@gmail.com'),('Ibrahim Jarif','jarifibrahim@gmail.com')]
+
```

Comment:
Extraneous change. Having two consecutive new lines is not a good practice. [[more read](http://pep8.org/#blank-lines)]

I think we should split the list something like this for better readability. [[more read](http://pep8.org/#maximum-line-length)]
```python
ADMINS = [
     ('Tapasweni Pathak','tapaswenipathak@gmail.com'),
     ('Nikhita Raghunath','nikitaraghunath@gmail.com'),
     ('Ibrahim Jarif','jarifibrahim@gmail.com')
]
```